README.md

```diff
@@ -81,18 +80,18 @@ as possible, and the following are the status:
 - [UDP/Datagram](docs/api/DGRAM.md)
 - [Zlib](docs/api/Zlib.md)
 
-In desktop and embedable ecosystem, the service via `DBus` is quite normal, in shadow-node, the `DBus`
-will be the built-in module. For detailed API, see [DBUS API](docs/api/DBUS.md).
+In desktop and embeddable ecosystem, the service via `DBus` is quite normal, in ShadowNode, the `DBus`
+will be a built-in module. For detailed API, see [DBus API](docs/api/DBUS.md).
```

Comment:
The DBus have a plan to be deprecated, so I think we could remove this description from README :)